Firestore/Example/App/iOS/Images.xcassets/AppIcon.appiconset/Contents.json

```diff
@@ -84,10 +84,15 @@
       "idiom" : "ipad",
       "size" : "83.5x83.5",
       "scale" : "2x"
+    },
+    {
+      "idiom" : "ios-marketing",
```

Comment:
Nit: are changes to this file intentional?

Reply:
Yes. They happen any time you manipulate the macOS project. I can put this in a separate PR if you prefer.